style.css

```diff
@@ -1,179 +1,400 @@
+  width: 327px;
+  gap: 120px;
+}
+
+.h {
+  gap: 40px;
+  margin-bottom: 40px;
+}
+
+.mainTxt-2 {
+  font-weight: 500;
+  font-size: 40px;
+  line-height: 52px;
+  color: #9490ff;
+  text-align: left;
+  width: 327px;
+}
+
+.workBlocks {
+  display: grid;
+  grid-template-rows: repeat(729px, 4);
+}
+
+.featuredImg {
+  flex: 0 0 729px;
+  max-width: 327px;
+  gap: 24px;
+  margin-bottom: 40px;
+}
+
+.imgPlaceHolder {
+  width: 327px;
+  height: 236px;
+  background: linear-gradient(0deg, #fff, #fff);
+}
+
+.rightBlock {
+  display: flex;
+  flex-direction: column;
+  max-width: 327px;
+  height: 469;
+}
+
+.titlePost {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: 24px;
+  text-align: left;
+}
+
+p.supportingTxt {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  align-self: left;
+}
+
+ul.tags {
+  justify-self: flex-start;
+  justify-content: flex-start;
+  margin: 0;
+  max-width: 312px;
+  padding: 0;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  list-style: none;
+  gap: 0;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.tags > li {
+  padding: 0;
+  padding: 10px 12px;
+  flex: 0 0 56px;
+  height: 40px;
+  border-left: 1px solid #555;
+  text-align: center;
+}
+
+.button.normal.preview,
+.button.normal.email {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  gap: 10px;
+  width: 89px;
+  height: 48px;
+  background: #7d73fc;
+  align-self: left;
+  border-radius: 0;
+  text-align: center;
+}
+
+.Page {
+  gap: 40px;
+}
+
+.cntcntHead {
+  width: 327px;
+  height: 164px;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact {
+  color: #9390ff;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  width: 327px;
+  margin-bottom: 24px;
+}
+
+.descript {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 40px;
+}
+
+form {
+  gap: 24px;
+}
+
+.field1 {
+  width: 327px;
+  height: 48px;
+  text-align: left;
+}
+
+.field2 {
+  width: 295px;
+  height: 12px;
+  text-align: left;
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #191765;
+  padding: 16px;
+}
+
+.field3 {
+  min-width: 327px;
+  min-height: 148px;
+  resize: none;
+  text-align: left;
+}
+
+.two {
+  margin-bottom: 0;
+}
+
+.e-address {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.button.normal.preview,
+.button.normal.email:hover {
+  background: #7d73fc;
+  box-shadow: 0px 8px 16px rgba(125, 115, 252, 0.48);
+}
+
+.button.normal.preview,
+.button.normal.email:active {
+  background-color: #5e42d7;
+  box-shadow: 0 8 16 rgba(125, 115, 252, 0.48);
+}
+
+.button.normal.preview,
+.button.normal.email:disabled {
+  background-color: #2a2b33;
+}
```

Comment:
Hi @Moathal ✋

- [ ] Based on what the previous reviewer pointed out. Try to implement the button interactions hover and enable as specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). Check the image left behind by the previous reviewer showing the enable and hover effects.